R/constants.R

```diff
@@ -0,0 +1,21 @@
+#' Platforms present in the R-hub repository.
+#' @keywords internal
+RHUB_REPO_PLATFORMS <- c(
+  "fedora-36",
+  "fedora-38",
+  "fedora-40",
+  "macos-arm64",
+  "macos-x86_64",
+  "ubuntu-22.04-aarch64",
+  "ubuntu-22.04-s390x",
+  "ubuntu-22.04",
+  "ubuntu-24.04-aarch64"
+)
+
+#' Base URL for the R-hub repository.
+#' @keywords internal
+RHUB_BASE_URL <- "https://raw.githubusercontent.com/r-hub/repos/main"
```

Comment:
I get your point, but I think we need to think more on who the user is in this case. The way I see it - the functionality that I'm developing in this PR will be only used in the github pipeline, not by individual people.

This constant variable is never used directly - only as a fallback if `PHARMAPKGS_REMOTE_REPO` environment variable is missing (makes it easier to test).

I think it's fine to keep it for now - it's mostly for our convenience, so that we have different hard-coded variables in a single place. Maybe we can later add another repo url here.